schema/src/main/scala/CpgExtSchema.scala

```diff
@@ -222,6 +222,10 @@ class CpgExtSchema(builder: SchemaBuilder, cpgSchema: CpgSchema) {
   highTouchSinkNode.addOutEdge(edge = taggedBy, inNode = tag)
 
   // HighTouch sink node - End
+
+  // Edge from derived source to the original source
+  private val originalSourceEdge = builder.addEdgeType(CpgSchemaConstants.ORIGINAL_SOURCE_EDGE_NAME)
+  astNode.addOutEdge(edge = originalSourceEdge, inNode = member)
```

Comment:
Derived source and be from any AST node, and not just Member

Reply:
Oops, missed that.